release_docs/USING_HDF5_CMake.txt

```diff
@@ -38,7 +38,7 @@ I. Preconditions
 ========================================================================
 
    1. We suggest you obtain the latest CMake for your platform from the Kitware
-      web site. The HDF5 1.13.x product requires a minimum CMake version
```

Comment:
More 1.15(1.13) should be 1.14

Reply:
We need to make these comments more maintainable